### DISCUSSION TITLE
About project development.

### DISCUSSION THREAD

**Discussion by ARDeepak** [github/india#358] [Students]
As there are cloud courses available to learn. with this we will understand about ,"How cloud computing works ." But to develop solution for a project we need coding . my question is, "How to represent solution?" . Is there any pre-code already present for such available projects or we need to develop our own code for the solution.

**Comment by TheSuranaverse** [comment]
A group for all to get notified for any **Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete** together.
One-stop place for daily **Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music** & much more...

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Reply by ghost** [reply]
The project seems to be a bit difficult task. There are no proper guidelines to complete project work. For students who are from non-CS backgrounds how can they code? How to write a project synopsis? what are the key steps to be followed?
